info.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# BEGIN LICENSE
+# Copyright (c) 2014 Jim Kemp <kemp.jim@gmail.com>
```

Comment:
For this file, strike this line. He had nothing to do with the info screen.

Reply:
done